.github/vcpkg.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "opensc",
```

Comment:
does this mean that we have an opensc vcspkg with this? would this be a new distribution path?

Reply:
No it is just manifest file to help install dependencies with vcpkg.